nave.sh

```diff
@@ -81,7 +81,7 @@ main () {
               )/$(basename -- "$SYM")
   done
 
-  if ! [ -d "$NAVE_DIR" ]; then
+  if ! [ -z "${NAVE_DIR+defined}" ]; then
```

Comment:
Seems like the logic is the wrong way around here – should get rid of the bang because `-z` checks if a variable is unset, so it should be: `if [ -z "${NAVE_DIR+defined}" ]`

Related: https://github.com/isaacs/nave/issues/81